staging/operator-lifecycle-manager/pkg/controller/registry/resolver/step_resolver_test.go

```diff
@@ -482,6 +489,7 @@ func TestResolver(t *testing.T) {
 			name: "SingleNewSubscription/ResolveOne/BundlePath",
 			clusterState: []runtime.Object{
 				newSub(namespace, "a", "alpha", catalog),
+				newOperatorGroup("foo", namespace),
```

Comment:
Had to had a few of these extra from the original commit. Looks like they were added in a different commit from Ankita but that was back ported till 4.14, so 4.13 was missing them and this commit needed them.